data/lib/src/network/model/request/download_request.dart

```diff
@@ -0,0 +1,61 @@
+// LinShare is an open source filesharing software, part of the LinPKI software
```

Comment:
Should I put this in Domain layer ?

Reply:
IMO, please add a new and clean `download request` model in domain (no serialization annotation), and make the mapper between this and that. @manhduc-nguyen please share your idea.